x-pack/plugins/security_solution/public/management/components/package_action_item/package_action_formatter.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import type { DocLinks } from '@kbn/doc-links';
+
+type PackageActions = 'es_connection';
+
+export const titles = Object.freeze(
+  new Map<PackageActions, string>([
+    [
+      'es_connection',
+      i18n.translate('xpack.securitySolution.endpoint.details.packageActions.es_connection.title', {
+        defaultMessage: 'Elasticsearch Connection Failure',
+      }),
+    ],
+  ])
+);
+
+export const descriptions = Object.freeze(
+  new Map<Partial<PackageActions> | string, string>([
+    [
+      'es_connection',
+      i18n.translate(
+        'xpack.securitySolution.endpoint.details.packageActions.es_connection.description',
+        {
+          defaultMessage:
+            "The Endpoint's connection to Elasticsearch is either down or misconfigured. Make sure it is configured correctly.",
```

Comment:
not sure if there were other discussions that happened later but my current understanding is that we can display `payload.error.message` directly for es connection errors since it'll be a user friendly message. though, not a big deal either way.

Reply:
I would say we want to use our custom translation for error messaging when it is possible so we can easily update/translate the message without changing endpoint response.
cc: @kevinlog